test/pseudo-tty/test-tty-get-color-depth.js

```diff
@@ -10,7 +10,7 @@ const writeStream = new WriteStream(fd);
 
 {
   const depth = writeStream.getColorDepth();
-  assert.equal(typeof depth, 'number');
+  assert.strictEqual(typeof depth, 'number');
```

Comment:
Can you also remove the eslint-disable comment on line 5? It's only purpose (at this point anyway) is to allow the file to pass linting even though it uses `assert.equal()` which is prohibited by our linter.

Reply:
Sure 👍